spring-cloud-sleuth-autoconfigure/pom.xml

```diff
@@ -414,6 +414,7 @@
 			<artifactId>zipkin</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<!-- Needed for ZipkinRestTemplateSenderConfiguration: BytesMessageEncoder  -->
```

Comment:
maybe this code should be rewritten to avoid classes or this dep become not optional?

```
backend         | 2024-01-08 05:03:14.185 [] [/] WARN  [main] o.s.c.s.AbstractApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'zipkinSender' defined in class path resource [org/springframework/cloud/sleuth/zipkin2/sender/ZipkinRestTemplateSenderConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [zipkin2.reporter.Sender]: Factory method 'restTemplateSender' threw exception; nested exception is java.lang.NoSuchMethodError: 'zipkin2.reporter.BytesMessageEncoder zipkin2.reporter.BytesMessageEncoder.forEncoding(zipkin2.codec.Encoding)'
```